modules/serverless-logic-upgrade-1-36-finalizing-data-index.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1-36-finalizing-data-index_{context}"]
+= Finalizing the Data Index upgrade
+
+After the Operator upgrade, if your deployment is configured to use a Knative Eventing Kafka Broker, you must delete the old `data-index-process-definition` trigger that was created in the {ServerlessLogicProductName} 1.35.0 version. Optionally, you can delete the old `ReplicaSet` resource as well.
+
+.Procedure
+
+. List all the triggers by running the following command:
++
+[source,terminal]
+----
+$ oc get triggers -n <target_namespace>
+----
++
+.Example output
+[source,sql,subs="verbatim,quotes"]
+----
+NAME                                                              BROKER              SUBSCRIBER_URI
+data-index-jobs-a25c8405-f740-47d2-a9a5-f80ccaec2955              example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/jobs
+data-index-process-definition-473e1ddbb3ca1d62768187eb80de99bca   example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/definitions
+data-index-process-error-a25c8405-f740-47d2-a9a5-f80ccaec2955     example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-instance-mul07f593476e8c14353a337590e0bfd5ae   example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-node-a25c8405-f740-47d2-a9a5-f80ccaec2955      example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-state-a25c8405-f740-47d2-a9a5-f80ccaec2955     example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-variable-487e9a6777fff650e60097c9e17111aea25   example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+
+jobs-service-create-job-a25c8405-f740-47d2-a9a5-f80ccaec2955      example-broker      http://sonataflow-platform-jobs-service.<target_namespace>.svc.cluster.local/v2/jobs/events
+jobs-service-delete-job-a25c8405-f740-47d2-a9a5-f80ccaec2955      example-broker      http://sonataflow-platform-jobs-service.<target_namespace>.svc.cluster.local/v2/jobs/events
+----
+
+. Based on the generated example output, delete the old `data-index-process-definition` trigger by running the following command:
++
+[source,terminal]
+----
+$ oc delete trigger data-index-process-definition-473e1ddbb3ca1d62768187eb80de99bca -n <target_namespace>
+----
++
+After deletion, a new trigger compatible with {ServerlessLogicProductName} 1.36.0 is automatically created.
+
+. (Optional) Identify the old `ReplicaSet` resource by running the following command:
```

Comment:
For all of these optional steps, per the ISG, they should be written as "Optional: "

```suggestion
. Optional: Identify the old `ReplicaSet` resource by running the following command:
```

Screenshot from ISG:

![image](https://github.com/user-attachments/assets/937c1075-7648-4748-b4c0-0fd544a0923f)